catalyst/callbacks/__init__.py

```diff
@@ -39,6 +40,12 @@
     LRFinder,
 )
 
+if SETTINGS.quantization_required:
+    from catalyst.callbacks.quantization import QuantizationCallback
+
+if SETTINGS.onnx_required:
+    from catalyst.callbacks.onnx import OnnxCallback
+
```

Comment:
could you please make import in alphabetical order of corresponding  `.py` files? (it's much easier to check imports correctness in this case)